srcpkgs/batsignal/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'batsignal'
+pkgname=batsignal
+version=1.1.2
+revision=1
+archs=noarch
+build_style=gnu-makefile
+hostmakedepends="pkg-config libglib-devel glib-devel"
+makedepends="libnotify-devel"
+short_desc="Simple battery monitor"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
+license="ISC"
+homepage="https://github.com/electrickite/batsignal"
+distfiles="https://github.com/electrickite/batsignal/archive/${version}.tar.gz"
+checksum=00cb30df33b5fee049963657f341e963a0f094dbf1620397bf1e56fa07191e7c
+
+post_install() {
+  make PREFIX=/usr DESTDIR="${DESTDIR}" install
```

Comment:
Also, `do_install` runs exactly this command, so you don't need it here. Just `vlicense LICENSE` is fine.